app/models/concerns/hyrax/in_admin_set.rb

```diff
@@ -5,5 +5,9 @@ module InAdminSet
     included do
       belongs_to :admin_set, predicate: ::RDF::Vocab::DC.isPartOf
     end
+
+    def active_workflow
+      Sipity::Workflow.find_active_workflow_for_admin_set_id(admin_set_id)
```

Comment:
How about using kwargs here and doing: `find_active_workflow_for(admin_set_id: admin_set_id)`